CannibalisticZombies/Assets/PlayerCharacterController.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```

Comment:
make sure to enclose every class in the namespace "CannibalisticZombies".

Ex:
![image](https://github.com/getmikyled/CannibalisticZombies/assets/128440175/2601fb0f-5dac-4a36-8e33-ae5f4ee4aeb7)

In this case, it would be `namespace CannibalisticZombies`